string name
float64 min_angle
float64 max_angle